README.md

```diff
@@ -27,23 +27,23 @@ This is the starting point for joining and contributing to the Elyra community -
 improving code, giving talks etc.
 
 
-### Dev Meetings
-Join us for our scrum to discuss development items you're working on or have questions about. Everyone is welcome and participation is optional. Note: Currently the Elyra community call is co-located with the CLAIMED community call.
+### Dev Meetings Elyra General
+Join us to discuss development items you're working on or have questions about. Everyone is welcome and participation is optional. Note: Currently the Elyra community call is co-located with the CLAIMED community call.
 
-**When**: Every Friday at [8:30AM Pacific](https://www.thetimezoneconverter.com/?t=8%3A30%20am&tz=San%20Francisco&)
+**When**: Every Friday at [8:30-9:00 AM Pacific](https://dateful.com/convert/san-francisco-california?t=0830)
 
-**Where**: [Webex](https://ibm.webex.com/meet/romeo.kienzler1)
+**Where**: [Zoom](https://zoom-lfx.platform.linuxfoundation.org/meeting/94379806729?password=f2c8ef12-518f-41ee-a64b-c169b9e9a2bb)
 
 **What**: [Current Milestone](https://github.com/elyra-ai/elyra/milestones)
 
-### Weekly Dev Community Meeting 
-Join us weekly to discuss Elyra development topics.  Everyone is welcome and participation is optional.
+### Weekly Dev Community Meeting Elyra Frontend
```

Comment:
are we having calls on Thursday? I don't think we need two calls per week, so I would just leave the Friday one.

Reply:
ah, yes, of course, both the main and the UI meeting are on Fridays, my error. Fixing it.

Reply:
so, the general meeting is Fridays at 8:30 to 9 am Pacific, the other one UI at 10 am Pacific, also Fridays.
@romeokienzler please have a look here, too, and update the email invites.